spring-cloud-dataflow-classic-docs/src/test/java/org/springframework/cloud/dataflow/server/rest/documentation/TaskSchedulerDocumentation.java

```diff
@@ -121,10 +121,10 @@ void listAllSchedules() throws Exception {
 				.andExpect(status().isOk())
 				.andDo(this.documentationHandler.document(
 						queryParameters(
-								parameterWithName("page")
-										.description("The zero-based page number (optional)"),
+								parameterWithName("page").optional()
+										.description("The zero-based page number"),
 								parameterWithName("size")
-										.description("The requested page size (optional)")),
+										.description("The requested page size")),
```

Comment:
We are missing the `.optional()` method here.

Reply:
Thanks. Fixed